### ISSUE TITLE
If only warnings are received, exit normally

### ISSUE THREAD

**Issue by konojunya** [markuplint/markuplint#1047]
If there is an error, exit code 1 is fine, but if it is warn only, it is easier to incorporate it into CI if you exit with exit code 0. 
Please let us know if there is some option to cover it, or if there is already an option.

like this https://eslint.org/docs/latest/use/command-line-interface#--max-warnings

```shell
markuplint index.html --max-warnings=-1
```

大規模なシステムに対して導入する際、どうしても一旦 warn にして CI に導入し少しずつ warn を減らしたり、数が少なくなったタイミングで default severity に従う形に変更するなどしたいです。そのため warn だけの場合には CI を通したいので exit code 0 になっていると助かるのですがそのようにできる option を新しく設ける、または既にあれば知りたいです

**Comment by YusukeHirao** [comment]
@konojunya 

最近、`--allow-warnings` というCLIオプションを追加しました。
（README.mdに無いのは書き忘れです）

https://github.com/markuplint/markuplint/pull/940

これで解決しませんか？

**Comment by konojunya** [comment]
@YusukeHirao  試してみます！ありがとうございます。一旦 close させていただきます！
